### PR TITLE
Fixes #8712 - added refresh_facts to api

### PR DESCRIPTION
**Pull request by orrabin**

### CODE REVIEW

File: app/controllers/api/v2/discovered_hosts_controller.rb [hunk 1]
```diff
@@ -106,6 +106,7 @@ def facts
         render :json => {'message'=>e.to_s}, :status => :unprocessable_entity
       end
 
+
```
Comment (ohadlevy) [inline app/controllers/api/v2/discovered_hosts_controller.rb]: nitpick: extra line

File: app/controllers/api/v2/discovered_hosts_controller.rb [hunk 2]
```diff
@@ -151,6 +152,15 @@ def auto_provision_all
         end
       end
 
+      api :PUT, "/discovered_hosts/:id/refresh_facts", N_("Refreshing the facts of a discovered host")
+      param :id, :identifier, :required => true
+
+      def refresh_facts
+        process_response @discovered_host.refresh_facts
+        rescue ::Foreman::Exception => e
```
Comment (ohadlevy) [inline app/controllers/api/v2/discovered_hosts_controller.rb]: nitpick: identation

**Comment by shlomizadok** [comment]
Tested and works well. @ohadlevy -- seems it's ready to merge.

**Comment by ohadlevy** [comment]
merged as 181d2b0 thanks @orrabin

**Pull request closed without merging**